source/Sashimi.Tests.Shared/Extensions/ValidatorExtensions.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentValidation;
+
+namespace Sashimi.Tests.Shared.Extensions
+{
+    public static class ValidatorExtensions
+    {
+        public static string[] ValidateAndGetErrors(this IValidator validator, object model)
+        {
+            var errors = validator.Validate(model).Errors.Select(e => e.ErrorMessage).ToArray();
+            foreach (var error in errors)
+            {
+                Console.WriteLine(error);
```

Comment:
Is this needed?

Reply:
It's nice to have when you're running the test